benchmarks/src/garage_benchmarks/run_benchmarks.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+"""Script for running benchmarking.
+
+Examples:
+    # List all benchmark options
+    benchmark show
+
+    # Run selected benchmarks
+    benchmark run b_1 b_2 ...
+
+"""
+import inspect
+
+import click
+from garage_benchmarks import benchmark_algos
+from garage_benchmarks import benchmark_auto
+from garage_benchmarks import benchmark_baselines
+from garage_benchmarks import benchmark_policies
+from garage_benchmarks import benchmark_q_functions
+
+
+@click.group()
+def cli():
+    """The main command group."""
+
+
+@click.command()
+def list():  # pylint: disable=redefined-builtin
+    """List all benchmarks."""
+    _echo_run_names('Algorithms', _get_runs_dict(benchmark_algos))
+    _echo_run_names('Policies', _get_runs_dict(benchmark_policies))
+    _echo_run_names('Baselines', _get_runs_dict(benchmark_baselines))
+    _echo_run_names('Q Functions', _get_runs_dict(benchmark_q_functions))
+    _echo_run_names('Automatic benchmarking', _get_runs_dict(benchmark_auto))
+
+
+@click.command()
+@click.argument('names', nargs=-1)
+def run(names):
+    """Run selected benchmarks.
+
+    Args:
+        names (tuple): Benchmark names.
+
+    Raises:
+        BadParameter: if any run name is invalid or duplicated.
+
+    """
+    if not names:
+        raise click.BadParameter('Empty names!')
+
+    if len(names) != len(set(names)):
+        raise click.BadParameter('Duplicate names!')
+
+    options = _get_all_options()
+
+    for name in names:
+        if name not in options:
+            raise click.BadParameter(
+                'Invalid run name! Make sure every name can be found in '
+                '`benchmark list`!')
```

Comment:
I guess this should be
```suggestion
                '`garage_benchmark list`!')
```